web/index.html

```
@@ -0,0 +1,46 @@
+<meta charset="utf-8">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)